docker-compose.yml

```diff
@@ -49,7 +49,8 @@ services:
       AIRFLOW_CONN_BOUWSTROOMPUNTEN_BASE_URL: "https://maps.nrgaccounting.nl"
       GOB_PUBLIC_ENDPOINT: "gob/graphql/streaming/"
       GOB_SECURE_ENDPOINT: "gob/secure/graphql/streaming/"
-      FERNET_KEY: ${FERNET_KEY}
+      FERNET_KEY: ${FERNET_KEY:-"YlCImzjge_TeZc7jPJ7Jz2pgOtb4yTssA1pVyqIADWg="}
```

Comment:
Misschien kan je nog tussen de configuratie opties `container_name` en `environment` de configuratie optie `depends_on` toevoegen.

```yaml
    depends_on:
      - database
      - dso_database
```

Hier liep is gisteren namelijk tegenaan. De `airflow` service wilde reeds zaken uitvoeren terwijl de database services nog niet gestart waren.

Reply:
Ik heb het toegevoegd, maar ben ik bang dat het helpt niet met de probleem dat `airflow` begint zijn dingen te doen voor dat de DB klaar is om connecties te accepteren.

Helaas `depends_on` wil niet meer wachten voor DB health checks: https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on